src/ontology/fbcv-edit.obo

```diff
@@ -9062,11 +9065,30 @@ creation_date: 2022-02-16T14:16:03Z
 id: FBcv:0009013
 name: chemical cell dissociation
 namespace: biosample_attribute
-def: "A sample in which a biological specimen has been separated into individual cells or a cell suspension by using non-enzymatic chemicals." [FBC:DPG]
+def: "A sample in which a biological specimen has been separated into individual cells by using non-enzymatic chemicals." [EFO:0011026]
+xref: EFO:0011026
 is_a: FBcv:0009010 ! cell dissociation
 created_by: http://orcid.org/0000-0002-6095-8718
 creation_date: 2022-02-16T14:19:01Z
 
+[Term]
+id: FBcv:0009014
+name: biosample treatment method
+namespace: biosample_attribute
+def: "Methods pertaining to the experimental treatments a biosample may have been subjected to before its collection." [FBC:DPG]
```

Comment:
I see that you have copied this from the parent class, but ideally a definition should be applicable to each child rather than describing the group of children (so should really be singular, as least for the first sentence). I also think that 'may have' is too weak and 'before its collection' could be confusing in some contexts (e.g. collecting organisms, then applying some experimental manipulation to them).

Maybe something like:
> Experimental treatment method that a biosample has been subjected to.


Incidentally, the methods themselves shouldn't be defined as samples/cell types... The more you look the more you find... (maybe don't worry about this for now).

Reply:
> Incidentally, the methods themselves shouldn't be defined as samples/cell types…

I agree, the definition should be about the actual _method_ and not about the _sample_ the method has been applied to. But since most terms under `biosample method` are defined that way, I am reluctant to change that.